recipe/meta.yaml

```diff
@@ -41,67 +37,73 @@ requirements:
     - m2-texinfo               # [win]
     - m2-curl                  # [win]
     - m2-p7zip                 # [win]
-    - readline                 # [not win]
-    - ncurses >=5.9*           # [not win]
+    - readline 6.2*            # [not win]
+    - ncurses 5.9*             # [not win]
     - gcc                      # [not win]
-    - jpeg 8d*                 # [not win]
+    - jpeg 9*                  # [not win]
     - curl                     # [not win]
-    - xz 5.2.*                 # [not win]
     - automake                 # [not win]
-    - pango                    # [linux]
     - openjdk
     - {{native}}pkg-config
-    - {{native}}bzip2          # [not osx]
+    - {{native}}bzip2 1.0.*    # [not osx]
     - {{native}}libjpeg-turbo  # [win]
     - {{native}}toolchain      # [win]
     - {{native}}libiconv       # [win]
     - {{native}}gmp            # [win]
     - {{native}}fftw           # [win]
-    - {{native}}xz             # [win]
+    - {{native}}xz 5.2.*
     - {{native}}mpfr           # [win]
     - {{native}}libsndfile     # [win]
     - {{native}}bwidget        # [win]
     - {{native}}tktable        # [win]
-    - {{native}}pcre           # [win]
+    - {{native}}pcre
+    - {{native}}icu 56.*       # [not win]
     - {{native}}pango 1.40.*   # [not win]
-    - {{native}}pcre 8.39*     # [not win]
-    - {{native}}icu            # [not win]
-    - {{native}}tk
-    - {{native}}zlib
+    - {{native}}tk 8.6.*       # [win]
+    - {{native}}tk 8.5.*       # [not win]
```

Comment:
The pinnings do break with some of the `m2w64-` counterparts, but I guess that won't be a problem as we can have `m2w64-tk` together with `tk`, right?